<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftLimitFormUi</class>
 <widget class="SlGfwDialogForm" name="SoftLimitFormUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>378</height>
   </rect>
  </property>
  <widget class="SlGfwLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>24</y>
     <width>171</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Siemens AD CH Simplified</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 255, 0)</string>
   </property>
   <property name="hmiFontSize">
    <number>-5</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="text">
    <string>电机软极限</string>
   </property>
  </widget>
  <widget class="SlGfwGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-8</x>
     <y>65</y>
     <width>301</width>
     <height>313</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="hmiFonts">
    <bool>true</bool>
   </property>
   <property name="hmiColors">
    <bool>true</bool>
   </property>
   <property name="navigationMode">
    <bool>true</bool>
   </property>
   <property name="hmiUseBigFont">
    <bool>false</bool>
   </property>
   <property name="hmiFontSize">
    <number>-5</number>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="SlGfwLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>0</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Siemens AD CH Simplified</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiColors">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="inverseColors">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>左铣削单元</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>47</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>横向电机上限位</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="LH_MotorLimitPosHighLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>47</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>47</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="LH_MotorLimitPosLowLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>79</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>77</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>横向电机下限位</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>77</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>垂向电机上限位</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="LV_MotorLimitPosHighLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="LV_MotorLimitPosLowLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>垂向电机下限位</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>横向进刀保护</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="LH_CutterProtectLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>垂向进刀保护</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="LV_CutterProtectLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>横向分步进刀值</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="H_CutBiasStepLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="H_MinCutStepLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>横向最小进刀量</string>
    </property>
   </widget>
  </widget>
  <widget class="SlGfwGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>65</y>
     <width>301</width>
     <height>313</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="hmiFontSize">
    <number>-5</number>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="SlGfwLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>0</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="inverseColors">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>右铣削单元</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>52</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>横向电机上限位</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="RH_MotorLimitPosHighLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>52</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Siemens AD CH Simplified</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="value">
     <string/>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>52</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="RH_MotorLimitPosLowLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>84</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>82</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>横向电机下限位</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>82</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>114</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>114</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>垂向电机上限位</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="RV_MotorLimitPosHighLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>114</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="RV_MotorLimitPosLowLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>143</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>143</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>垂向电机下限位</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>143</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="RH_CutterProtectLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>173</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>171</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>171</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>横向进刀保护</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="RV_CutterProtectLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>202</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>垂向进刀保护</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>垂向分步进刀值</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="V_CutBiasStepLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>232</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="V_MinCutStepLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>262</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="hmiFonts">
     <bool>true</bool>
    </property>
    <property name="hmiFontSize">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>垂向最小进刀量</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="hmiFontSize">
     <number>-5</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlGfwDialogForm</class>
   <extends>QWidget</extends>
   <header>slgfwdialogform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlGfwGroupBox</class>
   <extends>QGroupBox</extends>
   <header>slgfwgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlGfwLabel</class>
   <extends>QFrame</extends>
   <header>slgfwlabel.h</header>
  </customwidget>
  <customwidget>
   <class>SlGfwLineEdit</class>
   <extends>QFrame</extends>
   <header>slgfwlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
